.kokoro/tests/run_single_test.sh

```diff
@@ -41,6 +41,13 @@ fi
 nox -s "$RUN_TESTS_SESSION"
 EXIT=$?
 
+# Inject region tag data into the test log
+XUNIT_PATH="$PWD/sponge_log.xml"
+XUNIT_TMP_PATH="$PWD/drift_tmp.xml"
```

Comment:
Can you use temp directory for XUNIT_TMP_PATH?

Reply:
Done for `XUNIT_TMP_PATH`, though I _think_ `XUNIT_PATH` needs to stay in `$PWD`.